pkg/events/core.go

```diff
@@ -12065,7 +12066,7 @@ var CoreEvents = map[ID]Definition{
 		params: []trace.ArgMeta{
 			{Type: "const char*", Name: "file_name"},
 			{Type: "const char*", Name: "path"},
-			{Type: "mode_t", Name: "mode"},
+			{Type: "umode_t", Name: "mode"},
```

Comment:
this kprobe is reading an umode_t and storing in a mode_t:

`mode_t mode = (unsigned short) PT_REGS_PARM2(ctx);`

I believe that it can be updated to umode_t as well, WDYT?

Reply:
It already is:
![image](https://github.com/user-attachments/assets/5d02130e-6c7c-4bfc-8903-15a34d614428)